wagtail/admin/templates/wagtailadmin/home/branding_welcome_title.html

```diff
@@ -0,0 +1,4 @@
+{% load i18n %}
+{% block branding_welcome %}
+    {% blocktrans trimmed %}Welcome to the {{ site_name }} Wagtail CMS{% endblocktrans %}
+{% endblock %}
```

Comment:
might be good to add a new line here and also configure your editor to add line breaks at the end of your file.